apps/web/src/components/Basenames/UsernameProfileSectionFrames/FrameListItem.tsx

```diff
@@ -43,6 +44,11 @@ export default function FrameListItem({ url }: { url: string }) {
       });
   }, [logEventWithContext, removeFrame, url]);
 
+  // Don't render the entire FrameListItem if the frame has an error
+  if (frameHasError) {
+    return null;
+  }
+
```

Comment:
why do you need the null returns on both `FrameListItem` and `Frame`? 

since `FrameListItem` is what is actually rendering the `Frame` components, can you omit from `Frame`?